dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/api/Connection.java

```diff
@@ -149,14 +154,17 @@ public void onGoaway(Channel channel) {
 
     public void onConnected(Channel channel) {
         this.channel.set(channel);
+        // This indicates that the connection is available.
+        this.connectedFuture.complete(new Object());
```

Comment:
replact `new Object()` with const instance maybe better?

Reply:
ok, i will modify it.